docs/sphinx/source/conf.py

```diff
@@ -56,11 +56,22 @@
     'sphinx_gallery.gen_gallery',
     'sphinx_toggleprompt',
     'sphinx_favicon',
+    'hoverxref.extension',
 ]
 
 mathjax3_config = {'chtml': {'displayAlign': 'left',
                              'displayIndent': '2em'}}
 
+# Enable hover tooltips
+hoverxref_auto_ref = True
+hoverxref_roles = [
+    'term',
+    'ref',
+    'py:func',
+    'py:class',
+    'py:meth',
```

Comment:
```suggestion
    'func',
    'class',
    'meth',
```

Reply:
In fact, IMO all listed elements could be put into the list: https://www.sphinx-doc.org/en/master/usage/domains/python.html#cross-referencing-python-objects

Reply:
I tried that in [6b59eb0](https://github.com/pvlib/pvlib-python/pull/2290/commits/6b59eb05ebca038e506d96d81ee5c6d4143ccbfc) but it did not work, that's why I then tried adding the `py`. Sorry I forgot to name that commit before pushing so it is not very clear

Reply:
hmm... I tried without the py in [f4275b8](https://github.com/pvlib/pvlib-python/pull/2290/commits/f4275b81c78dace07264493d2f95e44e6e19f3e6) but it did not work, that's why I then tried adding the py. Sorry I forgot to name that commit before pushing so it is not very clear

Reply:
hmm... I tried without the py in [f4275b8](https://github.com/pvlib/pvlib-python/pull/2290/commits/f4275b81c78dace07264493d2f95e44e6e19f3e6) but it did not work, that's why I then tried adding the py

Reply:
oh actually I did not have `hoverxref_auto_ref = True` as well that time

Reply:
hmm... this did not work

Reply:
Ups, that's right. Have a look at the following configuration variable, it seems to be required https://sphinx-hoverxref.readthedocs.io/en/latest/configuration.html#confval-hoverxref_role_types

For reference, I'm using the PyBaMM PR linked here https://github.com/pvlib/pvlib-python/pull/2290#issuecomment-2466210862 . You can compare what other changes they did.

Reply:
Regarding the `py`, try using [`hoverxref_domains`](https://sphinx-hoverxref.readthedocs.io/en/latest/configuration.html#confval-hoverxref_domains) (`:py:` is a sphinx domain)